cfgov/ask_cfpb/tests/test_blocks.py

```diff
@@ -72,25 +83,50 @@ def test_how_to_block_renders_schema(self):
                 'step_content': 'Step content'
             }]
         }
-        expected_html = (
-            '<div itemscope'
-            '     itemtype="http://schema.org/HowTo"'
-            '     class="schema-block schema-block__how-to">'
-            '<h2 itemprop="name" class="schema-block_title">test title</h2>'
-            '<div itemprop="description" class="schema-block_description">'
-            '<div class="rich-text">test description</div>'
-            '</div>'
-            '<ol>'
-            '<li itemprop="step"'
-            '     itemscope'
-            '     itemtype="http://schema.org/HowToStep"'
-            '     class="schema-block_item">'
-            '<h3 itemprop="name" class="h4">Step one</h3>'
-            '<div itemprop="text">Step content</div>'
-            '</li>'
-            '</ol>'
-            '</div>'
-        )
```

Comment:
Ah I also see what's going on here — we're losing `<div class="rich-text">` around `test description` here too —I was thinking it's whitespace changes, but then [`self.assertHTMLEqual`](https://docs.djangoproject.com/en/2.2/topics/testing/tools/#django.test.SimpleTestCase.assertHTMLEqual) is supposed to ignore whitespace around HTML tags.

Reply:
This is a [deliberate change in Wagtail 2.10](https://docs.wagtail.io/en/v2.10/releases/2.10.html#div-class-rich-text-wrappers-removed-from-rich-text). It's possible to at least partially restore the old behavior with the provided [legacy rich text tag](https://docs.wagtail.io/en/v2.10/reference/contrib/legacy_richtext.html#legacy-richtext), but I don't know if this will fix all occurrences (for example, if we manually import and run that method anywhere).

I do think it would be good if we didn't have this div wrapper in our output (see rationale and discussion at https://github.com/wagtail/wagtail/issues/1214), but removing it might have _visual_ implications based on CSS rules.

Reply:
Ok. So, I just think I'd like to confirm, @cwdavies, that there is no visual difference between Wagtail 2.9 and 2.10 that doesn't require any CSS changes. Can you confirm that?

Reply:
We have the following CSS and JavaScript that looks for that `.rich-text` class:

https://github.com/cfpb/cfgov-refresh/blob/18b75f5c37afe1495c02054c6b16a742a3a3c1ce/cfgov/unprocessed/css/organisms/info-unit-group.less#L49-L56

https://github.com/cfpb/cfgov-refresh/blob/18b75f5c37afe1495c02054c6b16a742a3a3c1ce/cfgov/unprocessed/css/pages/ask.less#L15-L26

https://github.com/cfpb/cfgov-refresh/blob/18b75f5c37afe1495c02054c6b16a742a3a3c1ce/cfgov/unprocessed/css/rich-text.less#L1-L22

https://github.com/cfpb/cfgov-refresh/blob/18b75f5c37afe1495c02054c6b16a742a3a3c1ce/cfgov/unprocessed/css/schema.less#L5-L7

https://github.com/cfpb/cfgov-refresh/blob/18b75f5c37afe1495c02054c6b16a742a3a3c1ce/cfgov/unprocessed/js/routes/on-demand/youth-employment-programs/buying-a-car/index.js#L30-L32

Reply:
As rich-text is defined in css and javascript I am adding `wagtail.contrib.legacy.richtext` to `INSTALLED_APPS`

Reply:
@cwdavies that sounds fine. 

@Scotchester Should we add a task to remove the CSS dependency on the `rich-text` class?

Reply:
Yes, I would very much like to rid ourselves of that wrapper and associated CSS/JS.

Reply:
A separate task is required to update CSS and JavaScript to remove dependency on the `rich-text` class